### ISSUE TITLE
Separate keyboard shortcuts for "Execute Query in Cursor Range" vs "Execute All Queries"

### ISSUE THREAD

**Issue by shadanan** [hankinsoft/SQLPro#1017]
Execute query in cursor range is great, and I'm glad that I can enable it as the default. But it would be really nice if there were separate shortcut keys for executing current SQL query, vs executing the whole SQL script:

<img width="936" alt="image" src="https://github.com/user-attachments/assets/defebd85-c399-469b-a6f3-9877dfd98668">

For example:
 - ⌘⇧E = Run All Queries
 - ⌘⏎ = Run Query in Cursor Range

So, in other words, the behavior of ⌘⇧E could remain unchanged, and we just add an additional shortcut key for executing query in cursor range..

**Environment**
 - Device: MacBook Pro 14-inch, 2021
 - OS: macOS 15.1.1 (24B91)
 - SQLPro app Version: SQLPro Studio Version 2024.60 (Build 111111)
- Installation source: App Store
- Target database server: PostgreSQL

**Comment by hankinsoft** [comment]
Could you try this build:
https://d3fwkemdw8spx3.cloudfront.net/studio/SQLProStudio.2024.74.app.zip

It adds 'Query execution (secondary)'
<img width="608" alt="Screenshot 2024-11-21 at 2 05 06 PM" src="https://github.com/user-attachments/assets/f36a6223-e19f-4f45-b638-07b0ac891b36">

Which will let you set a shortcut for the secondary query option. If you get a chance, please let me know if that works as you would expect, and if so, I will get the App Store version updated as well.

**Comment by shadanan** [comment]
Thanks for the super fast turnaround. That's kind of amazing.

I gave the app a try, and I think it works great and I'm happy with it as is.

For new users though, the discoverability of this feature may be problematic, and the behavior may be surprising. For me, it took a while to discover the "Execute query in cursor range" feature. My suggestions to improve usability:

1. Remove the "Execute query in cursor range" setting, and just make it "Enabled (but not default)" by default.
    <img width="491" alt="image" src="https://github.com/user-attachments/assets/8145deeb-dac5-45a2-b76b-b989e880e2d0">
2. The naming of the shortcuts are currently confusing (Query execution / Query execution (secondary)):
    <img width="432" alt="image" src="https://github.com/user-attachments/assets/5dcbce44-d8bd-4d0a-95c6-3cf6fcde6c53">
    Now that I've played with the build you sent me, I understand why they got named that way. But, if we remove the "Execute query in cursor range" setting, then the names of the shortcuts could be a lot less confusing:
  - Replace "Query execution" with "Run all / Run selected"
  - Replace "Query execution (secondary)" with "Run current query"
3. Instead of a dropdown button with multiple ways to execute:
    <img width="212" alt="image" src="https://github.com/user-attachments/assets/3fce3f71-6a2a-4381-a2f6-553989c07b22">
    You could just have two buttons:
    - "Run all / Run selected" (depending on whether something is selected or not)
    - "Run current query"

I'm otherwise totally happy with the current implementation! And the only reason I'm making such a detailed suggestion is because you seem to be listening. No good deed goes unpunished, or whatever. Thanks so much for such an awesome product.
